### DISCUSSION TITLE
You can’t perform that action at this time. Hatası

### DISCUSSION THREAD

**Discussion by Zeynel26477575** [IBAX-io/go-ibax#871] [Soru Cevap - Türkçe] [answered]
Star a basınca " You can’t perform that action at this time. "  deniyor. neden?

**Comment by Zeynel26477575** [comment, marked as answer]
I choosed  
✨ Inspiration
🚀 My stack
🔮 Future ideas
and problem solved.

**Reply by Yusufknk** [reply]
Yoğunluk dolayı daha sonra tekrar deneyin ya da farklı bir tarayıcı kullanın.

**Reply by Zeynel26477575** [reply]
Teşekkürler.
